scripts/build-prep-1node-cortx-mgw.sh

```diff
@@ -88,10 +88,11 @@ echo '[client.rgw.8000]'
 echo '      ...'
 echo '      motr my endpoint  = inet:tcp:10.0.0.1@5001'
 echo '      motr my fid       = 0x7200000000000001:0x29'
-echo 'Then run:  MDS=0 RGW=1 ../src/vstart.sh -d $mgs'
+echo "Current directory: $PWD"
+echo "Then run:  MDS=0 RGW=1 ../src/vstart.sh -d $mgs"
```

Comment:
Do you have `msg` variable to show here?

Reply:
yes.

Reply:
yes, in this script mgs is set to --without-dashboard" by default

Reply:
ok